primitives/state-machine/src/stats.rs

```diff
@@ -29,7 +31,13 @@ pub struct UsageUnit {
 	pub bytes: u64,
 }
 
+/// Ignore stat for no_std.
+#[cfg(not(feature = "std"))]
+#[derive(Clone, Debug, Default)]
+pub struct UsageUnit;
+
 /// Usage statistics for state backend.
+#[cfg(feature = "std")]
```

Comment:
Can we not just make `started` and maybe `span` optional on `no_std`? 

So we don't require all this special implementations here?

Reply:
I did transform that to a little test, nothing really fancy, I guess fancy test would be cumulus side (the branch https://github.com/paritytech/cumulus/compare/ec-witness-ext was passing test but I am not sure if there is much).